### DISCUSSION TITLE
python 網頁自動化及爬蟲

### DISCUSSION THREAD

**Discussion by Sheng-Hong-Wang** [1101-GS3073-C/Python-Community-GS3073-C#37] [General]
前幾天因youtube演算法推薦滑到用python實現網頁自動化和爬蟲的入門影片，覺得挺酷的！在這邊分享給各位
影片中講到一些基本的爬蟲技巧，如：自動點擊網頁中的標籤(按鈕，文字等)、蒐集標籤文字及儲存圖片等
而環境只需下載ChromeDriver和selenium，且下載過程非常容易，推薦大家有空玩玩看
(前陣子popcat很紅，利用這個影片所教的內容便能很輕鬆地刷點擊了)

ChromeDriver下載網址：https://sites.google.com/a/chromium.org/chromedriver/downloads
selenium下載：於終端機輸入pip install selenium
影片網址：https://www.youtube.com/watch?v=ximjGyZ93YQ&list=LL&index=14&t=282s

popcat code:
```
from selenium import webdriver

PATH = "D:/chromedriver.exe"
driver = webdriver.Chrome(PATH)
driver.get("https://popcat.click/")

while True:
    clickcat = driver.find_element_by_class_name("cat-img")
    clickcat.click()
```

**Comment by jim11261314** [comment]
這是一個很好的資料來源，我目前也才剛理解何謂爬蟲，並嘗試寫出相關程式碼。
